lib/msgapi/message.go

```diff
@@ -35,12 +35,20 @@ func (m *Message) ParseRaw() error {
       m.kludges["TOPT"]=l[6:]
     } else if len(l)>5 && l[0:6]=="\x01FMPT " {
       m.kludges["FMPT"]=l[6:]
+    } else if len(l)>8 && l[0:9]=="\x01REPLYTO " {
+      m.kludges["REPLYTO"]=l[9:]
+    } else if len(l)>7 && l[0:8]=="\x01MSGID: " {
+      m.kludges["MSGID"]=l[8:]
     }
   }
   log.Printf("%#v", m.kludges)
   if _, ok := m.kludges["INTL"]; ok {
     m.ToAddr=types.AddrFromString(strings.Split(m.kludges["INTL"]," ")[0])
     m.FromAddr=types.AddrFromString(strings.Split(m.kludges["INTL"]," ")[1])
+  } else if _, ok := m.kludges["REPLYTO"]; ok {
+    m.FromAddr=types.AddrFromString(strings.Split(m.kludges["REPLYTO"]," ")[0])
```

Comment:
only if m.FromAddr=nil.

Reply:
Is it needed here?
As far as I understand "else if" logic 2nd branch would be triggered only if no INTL kludge (and so m.FromAddr is nil) and then 3rd one if no INTL nor REPLYTO kludge are in the map (and so m.FromAddr still nil).